src/app/qgsvectorlayerproperties.cpp

```diff
@@ -293,7 +293,6 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
   );
 
   QString myStyle = QgsApplication::reportStyleSheet();
-  myStyle.append( QStringLiteral( "body { margin: 10px; }\n " ) );
```

Comment:
I though it's better to have it centralized in the main CSS stylesheet.
I'm working on the raster dialog and I though it's better to have this CSS in one place only.

I'm happy to rollback if needed.